testsuite/tests/system/messages/test_automatic_emails.py

```diff
@@ -0,0 +1,70 @@
+"""
+Rewrite /spec/functional_specs/account_message_spec.rb
+When sending message to a developer account, the email is send
+"""
+import time
+
+import pytest
+
+from testsuite import rawobj
+from testsuite.utils import blame
+
+
+@pytest.fixture()
+def params(request):
+    """
+    :return: params for custom account
+    """
+    name = blame(request, "id")
+
+    return {"name": name, "username": name, "org_name": name, "email": f"{name}@anything.invalid"}
+
+
+@pytest.fixture()
+def custom_app(custom_application, custom_app_plan, service, lifecycle_hooks, request):
+    """
+    Create a application without cleanup
+    """
+    plan = custom_app_plan(rawobj.ApplicationPlan(blame(request, "app_plan")), service)
+    return custom_application(
+        rawobj.Application(blame(request, "app_name"), plan), autoclean=False, hooks=lifecycle_hooks
+    )
+
+
+@pytest.fixture()
+def service_settings(request):
+    """
+    Makes service with required name
+    """
+    return {"name": blame(request, "svc")}
+
+
+@pytest.fixture()
+def only_service(custom_service, service_settings):
+    """
+    Create a custom service without backend and cleanup
+    """
+    return custom_service(service_settings, autoclean=False)
+
+
+def test_account_deleted_automatic_email(mailhog_client, custom_account, params):
+    """
+    Create new account
+    Delete account
+    Assert that a delete email has been sent
+    """
+    account = custom_account(params, autoclean=False)
+    account.delete()
+    mailhog_client.assert_message_received(subject=f"Account {account.entity_name} deleted", expected_count=1)
+
+
+def test_service_deleted_automatic_email(mailhog_client, only_service):
```

Comment:
not necessary to change anything, just a comment. These two tests seem to share nothing. In such case having them in separate files would make much clearer which fixture belongs to which test.